src/ServiceManager.php

```diff
@@ -638,52 +630,6 @@ private function doCreate(string $resolvedName, ?array $options = null)
         return $object;
     }
 
-    /**
```

Comment:
IMO worth checking proxy-manager-lts meanwhile: I didn't touch `ocramius/proxy-manager` in over 2 years, since the appearance of `proxy-manager-lts` removed all will of mine to maintain the original `proxy-manager`.

In other words, `proxy-manager-lts` makes my blood boil, but it is better to introduce it here than removing these critical tests and code here.

Reply:
I feel like it shouldn't be `laminas/laminas-servicemanager` responsibility to provide this functionality *out-of-the-box*, and since the first release supporting lazy services 8 years ago, lazy loading is now *built-in* in pretty much every library I use nowadays.

But if you want me to replace `ocramius/proxy-manager` with `friendsofphp/proxy-manager-lts` instead of dropping it, ping me again and I'll do it.

Reply:
Yes please

Reply:
Done.